modules/install-consul/install-consul

```diff
@@ -334,7 +334,12 @@ function install {
     install_tls_certificates "$path" "$user" "$ca_file_path" "$cert_file_path" "$key_file_path"
   fi
 
-  log_info "Consul install complete!"
+  if command -v consul; then
+    log_info "Consul install complete!";
+  else
+    log_info "Consul was not installed properly.  Aborting.";
```

Comment:
Add "Could not find consul command" to make it clear what about the install failed.

Reply:
One other thought: check if the `install-consul` script logs the download URL anywhere...